controllers/testutils/agent.go

```diff
@@ -344,9 +344,12 @@ func NewDatadogAgentWithGlobalConfigSettings(namespace string, name string) v2al
 				AppKey: apiutils.NewStringPointer("my-app-key"),
 			},
 		},
-		Registry:              apiutils.NewStringPointer("my-custom-registry"),
-		LogLevel:              apiutils.NewStringPointer("INFO"),
-		Tags:                  []string{"tagA:valA", "tagB:valB"},
+		Registry: apiutils.NewStringPointer("my-custom-registry"),
+		LogLevel: apiutils.NewStringPointer("INFO"),
+		Tags:     []string{"tagA:valA", "tagB:valB"},
+		OriginDetectionUnified: &v2alpha1.OriginDetectionUnified{
+			Enabled: apiutils.NewBoolPointer(false),
+		},
```

Comment:
Do we need this change? Or should we change a test to validate that the envvar was added properly to the `agent` container? where the dogstatsd server runs?

Reply:
I have just removed it 👍